requirements.txt

```diff
@@ -1,2 +1 @@
-Django>=1.8
```

Comment:
I never thought about people not using setuptools, but agreed on that point, and agreed that using auditlog it seems pretty obvious you would want a supported version of Django to run it :)